crates/store/src/errors.rs

```diff
@@ -117,105 +111,105 @@ impl From<DatabaseError> for Status {
 
 #[derive(Error, Debug)]
 pub enum StateInitializationError {
-    #[error("Database error: {0}")]
+    #[error("database error")]
     DatabaseError(#[from] DatabaseError),
-    #[error("Failed to create nullifier tree: {0}")]
-    FailedToCreateNullifierTree(NullifierTreeError),
-    #[error("Failed to create accounts tree: {0}")]
-    FailedToCreateAccountsTree(MerkleError),
+    #[error("failed to create nullifier tree")]
+    FailedToCreateNullifierTree(#[from] NullifierTreeError),
+    #[error("failed to create accounts tree")]
+    FailedToCreateAccountsTree(#[from] MerkleError),
 }
 
 #[derive(Debug, Error)]
 pub enum DatabaseSetupError {
-    #[error("I/O error: {0}")]
+    #[error("I/O error")]
     IoError(#[from] io::Error),
-    #[error("Database error: {0}")]
+    #[error("database error")]
     DatabaseError(#[from] DatabaseError),
-    #[error("Genesis block error: {0}")]
+    #[error("genesis block error")]
     GenesisBlockError(#[from] GenesisError),
-    #[error("Pool build error: {0}")]
+    #[error("pool build error")]
     PoolBuildError(#[from] deadpool_sqlite::BuildError),
-    #[error("SQLite migration error: {0}")]
+    #[error("SQLite migration error")]
     SqliteMigrationError(#[from] rusqlite_migration::Error),
 }
 
 #[derive(Debug, Error)]
 pub enum GenesisError {
     // ERRORS WITH AUTOMATIC CONVERSIONS FROM NESTED ERROR TYPES
     // ---------------------------------------------------------------------------------------------
-    #[error("Database error: {0}")]
+    #[error("database error")]
     DatabaseError(#[from] DatabaseError),
-    #[error("Block error: {0}")]
+    #[error("block error")]
     BlockError(#[from] BlockError),
-    #[error("Merkle error: {0}")]
+    #[error("merkle error")]
     MerkleError(#[from] MerkleError),
```

Comment:
Might be worthwhile to check if any of these could be made more concrete as well.

Reply:
They probably can; but I think I'll yank the entire error enum instead. There isn't really any recourse to failure of database setup.